pontoon/insights/static/js/insights_charts.js

```diff
@@ -36,52 +36,15 @@ var Pontoon = (function (my) {
         });
 
         // Set up default Chart.js configuration
-        Chart.defaults.global.defaultFontColor =
-          style.getPropertyValue('--light-grey-7');
-        Chart.defaults.global.defaultFontFamily = 'Open Sans';
-        Chart.defaults.global.defaultFontStyle = '100';
-        Chart.defaults.global.datasets.bar.barPercentage = 0.7;
-        Chart.defaults.global.datasets.bar.categoryPercentage = 0.7;
-      },
-      // Legend configuration doesn't allow for enough flexibility,
-      // so we build our own legend
-      // eslint-disable-next-line no-unused-vars
-      customLegend: (chart) => (chart) => {
```

Comment:
Why don't we replace this with the contents of the generalized `htmlLegendPlugin`, which is currently defined in two different places, and thus more prone to errors than if it were kept in one place?

Reply:
This one still needs to be addressed.